runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -2375,6 +2375,112 @@ ClassFileOracle::methodIsNonStaticNonAbstract(U_16 methodIndex)
 	return J9_ARE_NO_BITS_SET(_classFile->methods[methodIndex].accessFlags, (CFR_ACC_STATIC | CFR_ACC_ABSTRACT));
 }
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+/**
+ * Determine if the method name corresponds to a VarHandle method with polymorphic
+ * signature.
+ *
+ * @param methodName the constant pool info for the method name
+ *
+ * @return true for a VarHandle method with polymorphic signature. Otherwise,
+ * return false.
+ */
+bool
+ClassFileOracle::isPolymorphicVarHandleMethod(J9CfrConstantPoolInfo *methodName)
```

Comment:
There's a very similar method in ConstantPoolMap that determines if a cpEntry is for a VarHandle method.  Is it possible to reuse the existing method - `ConstantPoolMap::isVarHandleMethod` - or share the list between the two methods?  

Duplicating these kinds of lists leads to one getting out of sync.

If the long term goal is to remove the other list, then adding comments to both lists referencing the other would help with short term maintenance.

Reply:
I did notice another usage of that list here: https://github.com/eclipse/openj9/blob/adda8032110a449a8c96bb1901c0996900555697/runtime/vm/initsendtarget.cpp#L34-L148

I was planning to share the list among all three methods.

Reply:
I was only able to share code between 
[1] `ConstantPoolMap::isVarHandleMethod` and 
[2] `ClassFileOracle::isPolymorphicVarHandleMethod`.

[3] `initsendtarget.cpp::initializeMethodRunAddressVarHandle` targets a different class (`VarHandleInternal`) and methods (with suffix `_impl`).